### ISSUE TITLE
Not compatible with newest version 1.6 :(

### ISSUE THREAD

**Issue by MrBlastman** [uberhalit/SekiroFpsUnlockAndMore#53]
Will not work with version 1.6.  Sad.  :(

**Comment by SetilaSilvios** [comment]
Just ignore the error popup.
Close the popup and press the Patch game button, then the program works well.

**Comment by MrBlastman** [comment]
> Just ignore the error popup. Close the popup and press the Patch game button, then the program works well.

You are right!  I needed this to patch in 32:9 super ultrawide, and after ignoring error I can then click the checkboxes, set resolution, and then click patch.

Thanks!

**Comment by plugpulled** [comment]
> Just ignore the error popup. Close the popup and press the Patch game button, then the program works well.

Game doesn't work properly. The HDR is broken when launching with this tool.

**Comment by doctorsangria** [comment]
> > Just ignore the error popup. Close the popup and press the Patch game button, then the program works well.
> 
> Game doesn't work properly. The HDR is broken when launching with this tool.

ive had problems with hdr both with and without using this tool, but i managed to solve it in nvidia control panel by going to display --> change resolution, scrolling down and selecting "use NVIDIA color settings" instead of "use default color settings"
